test/e2e/toolchain_test.go

```diff
@@ -55,28 +57,29 @@ func TestToolchain(t *testing.T) {
 
 		err = await.WaitForCheInstallConditions(cheInstallation.Name, wait.UntilHasCheStatusCondition(cheinstallation.SubscriptionCreated()))
 		require.NoError(t, err)
-		checkCheResources(t, f.Client.Client, cheOperatorNS, cheOg, cheSub)
+		checkCheResources(t, f.Client.Client, cheOperatorNS, cheOg, cheSub, cheCluster)
 	})
 
-	t.Run("should recreate che operator's ns operatorgroup subscription when ns deleted", func(t *testing.T) {
-		// given
-		ns := &v1.Namespace{}
-		err := f.Client.Get(context.TODO(), types.NamespacedName{Name: cheOperatorNS}, ns)
-		require.NoError(t, err)
+	// TODO enable this test. The issue is Namespace when deleted, stuck in Terminating Phase
```

Comment:
let's create a Jira issue for this

Reply:
@MatousJobanek if you have created Jira issue then kindly add link here.  I guess @xcoulon is actively looking into the scenario when namespace status is stuck with `Terminating` status.